File: Java/URI1009.java
import java.util.Scanner;

public class URI1009 {
	public static void main(String[] arg) {
		Scanner entrada = new Scanner(System.in);
		String nome = entrada.next();
		double salario = entrada.nextDouble();
		double vendas = entrada.nextDouble();
		double salarioTotal = salario + (vendas * 0.15);
		System.out.printf("TOTAL = R$ %.2f\n", salarioTotal);
		
	}

}

File: C/uri1009.c
#include <stdio.h>

void main(){
    char nome[50];
    double salario, vendas;
    scanf("%s %lf %lf", nome, &salario, &vendas);
    printf("TOTAL = R$ %.2lf\n", (salario + 0.15*vendas));
}